packages/app/scripts/downloadHelpItems.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+const axios = require("axios")
+const path = require("path")
+const fs = require("fs-extra")
+require("dotenv").config({
+  path: path.join(__dirname, "../../../.env"),
+})
+import { normalizeHelpData } from "../../app/src/__deprecated__/renderer/utils/contentful/normalize-help-data"
+;(async () => {
+  try {
+    await fs.ensureDir(path.resolve(path.join("src", "main")))
+    const jsonPath = path.join("src", "help", "default-help.json")
+
+    const url = `${process.env.MUDITA_CENTER_SERVER_URL}/help`
+    const { data } = await axios.get(url, {
+      params: { limit: 6 },
```

Comment:
Why 6? If that's because the server uses 6 as the default value for the limit variable maybe we can leave that decision to endpoint logic.

Reply:
I basically used news download script and modified it. That is something I've missed. It may also mean that the news script should not use limit param either🤔.

Reply:
I've removed the limit param as I didn't see any changes to the result with or without it